src/lib/comms/include/sol-network.h

```diff
@@ -73,6 +73,9 @@ extern "C" {
  */
 #define SOL_INET_ADDR_STRLEN 48
 
+#ifndef AF_BT_IOTIVITY
+    #define AF_BT_IOTIVITY 31
```

Comment:
You're right this is Zephyr case only. It won't work for Linux, at least for now.

I should add:
`#ifdef SOL_PLATFORM_ZEPHYR`

What do you think ?